types/chrome-remote-interface/index.d.ts

```diff
@@ -45,6 +45,18 @@ declare namespace CDP {
         sessionId?: string | undefined;
     }
 
+    interface SendError {
+        code: number;
+        message: string;
+        data?: string | undefined;
+    }
+
+    interface SendCallback<T extends keyof ProtocolMappingApi.Commands> {
+        (error: true, response: SendError): void;
+        (error: false, response: ProtocolMappingApi.Commands[T]['returnType'] | {}): void;
```

Comment:
```suggestion
        (error: false, response: ProtocolMappingApi.Commands[T]['returnType']): void;
```

Sorry the `{}` was wrong as per https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/ban-types.md#default-options. My bad.

I think it should be rare that the callback will get an empty object by chance nowadays since [node-inspector](https://github.com/node-inspector/node-inspector) is mostly deprecated so I think `{}` should *not* be replaced by `Record<string, never>`.